.github/workflows/dart.yml

```diff
@@ -39,6 +40,15 @@ jobs:
         shell: bash
       - name: Analyze code
         run: dart analyze --fatal-infos .
+      - name: Check that grpc-web sample builds with DDC
```

Comment:
Not so familiar with those `dart.yml` files, but it seems a little strange to have building an app as part of the `analyze: ` task.

Reply:
Yeah, it is a bit strange, but we use compilation as a way to force dependency checking here (similar to running `dart analyze`).